src/game/models/asteroid.rs

```diff
@@ -6,7 +6,8 @@ use piston_window::{Context, polygon, Size, Transformed, UpdateArgs};
 use rand;
 
 use super::super::color;
-use super::{Collidable, Drawable, PI_MULT_2, Positioned, Updateable, Vector};
+use super::{Collidable, Drawable, PI_MULT_2, Positioned, Updateable};
```

Comment:
Not needed for this PR, but I think we should consider ditching all of the `super::`'s and just use absolute imports for everything.

See: https://github.com/rust-lang-nursery/fmt-rfcs/issues/24#issuecomment-254353811